src/swagger.ts

```diff
@@ -39,6 +39,8 @@ export const SwaggerDocs = (app: INestApplication) => {
   );
 
   const options = {
+    // Swagger introduces custom URLs for JSON export https://github.com/nestjs/swagger/pull/2273
+    jsonDocumentUrl: 'openapi/json',
```

Comment:
Isn't this `openapi.json`?

Reply:
It can be any link, I decided to follow modern URL convention instead of hard extensions.
We can all further customise the above if requirement arises for a specific JSON export i.e, `currency/openapi/json`